src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -531,6 +531,23 @@ function renderSubtreeIntoContainer(
   );
 
   if (__DEV__) {
+    let warned;
+    if (!warned && container._reactRootContainer) {
+      const firstChild = container.firstChild;
+      const hostInstance =
+        DOMRenderer.findHostInstance(container._reactRootContainer.current);
+      if (hostInstance) {
+        warned = true;
```

Comment:
this variable doesn't do anything? I think it's fine to deduplicate but also fine not to